docs/stdlib/json.rst

```diff
@@ -132,26 +132,24 @@ possible to cast a JSON value directly into a :eql:type:`tuple`.
         db> select <bool>to_json('true');
         {true}
 
-    Note that a :eql:type:`json` value can be cast into a :eql:type:`str`
-    only when it is a JSON string.  Therefore, while the following will work
-    as expected:
+    A :eql:type:`json` type can also be casted as a :eql:type:`str` type,
+    but only when recognised as a JSON string:
 
     .. code-block:: edgeql-repl
 
         db> select <str>to_json('"something"');
         {'something'}
 
-    The operation below (casting a JSON array of
-    string ``["a", "b", "c"]`` to a *str*) will result in an error:
+    Casting a string array of JSON (``["a", "b", "c"]``) to a :eql:type:`str`
+    type will result in an error:
 
     .. code-block:: edgeql-repl
 
         db> select <str>to_json('["a", "b", "c"]');
         InvalidValueError: expected json string or null; got JSON array
 
-    Use the :eql:func:`to_json` and :eql:func:`to_str`
-    functions to dump or parse a :eql:type:`json` value to or
-    from a :eql:type:`str`:
+    For dumping or parsing the value of a :eql:type:`json` or :eql:type:`str`
+    type, use the :eql:func`to_json` or :eql:func:`to_str` functions:
```

Comment:
Split these two functions into their own sentences to avoid all the "or"s.
```rest
    Instead, use the :eql:func:`to_str` function to dump a JSON value to a
    :eql:type:`str` value. Use the :eql:func`to_json` to parse a JSON string to
    a :eql:type:`json`  value.
```